packages/angular/src/migrations/update-15-9-0/update-testing-tsconfig.spec.ts

```diff
@@ -75,6 +76,18 @@ async function setup(tree: Tree, name: string) {
     skipPackageJson: true,
   });
 
+  const projectConfig = readProjectConfiguration(tree, name);
+  updateProjectConfiguration(tree, name, {
+    ...projectConfig,
+    targets: {
+      ...projectConfig.targets,
+      test: {
+        ...projectConfig.targets.test,
+        executor: '@nrwl/jest:jest',
```

Comment:
Should this not be `@nx/jest:jest`?

If someone is _only_ migrating to NX 15.9.0 - the Nx 15.9.0 release will have a version of this migration that uses the @nrwl/ scoped packages.

This change will only be applied to people migrating from Nx < 15.9.0 to Nx latest

Reply:
This should be `@nrwl/jest:jest`. 

The migrations will run in order, so this will work on anything that's `@nrwl/jest:jest` and then the `16.0.0` migration will change them to `@nx/jest:jest`.